src/main/java/org/sagebionetworks/bridge/hibernate/HibernateStudyActivityEventDao.java

```diff
@@ -31,8 +31,8 @@
     static final String GET_RECENT_SQL = "SELECT *, (SELECT count(*) as total FROM " +
             "StudyActivityEvents WHERE eventId = sae.eventId AND studyId = :studyId " +
             "AND userId = :userId GROUP BY eventId) FROM StudyActivityEvents AS sae " +
-            "WHERE userId = :userId AND studyId = :studyId AND createdOn = (SELECT " +
-            "createdOn FROM StudyActivityEvents WHERE userId = :userId AND studyId = " +
+            "WHERE userId = :userId AND studyId = :studyId AND eventTimestamp = (SELECT " +
+            "eventTimestamp FROM StudyActivityEvents WHERE userId = :userId AND studyId = " +
             ":studyId AND eventId = sae.eventId ORDER BY createdOn DESC LIMIT 1) " +
             "ORDER BY eventId";
```

Comment:
Where's the test for this?

Reply:
The test refers directly to this SQL constant so when this changes, the test always passes. Which makes sense.